tests/unit/ml_handlers/test_autogluon.py

```diff
@@ -0,0 +1,138 @@
+                for col in ret.columns
+            ]
+            return pd.DataFrame(ret.data, columns=columns)
+
+    @patch('mindsdb.integrations.handlers.postgres_handler.Handler')
+    def test_simple(self, mock_handler):
+
+        self.set_handler(mock_handler, name='pg', tables={'df': self.df})
+
+        # create predictor
+        self.run_sql('''create model proj.modelx from pg (select * from df) predict c using engine='autogluon',
+        tag = 'test_model';''')
+        self.wait_predictor('proj', 'modelx')
+
+        # run predict
+        ret = self.run_sql('''
+           SELECT p.*
+           FROM pg.df as t
+           JOIN proj.modelx as p
+           where t.c=1
+        ''')
+        avg_c = pd.to_numeric(ret.c).mean()
+        # value is around 1
+        assert (avg_c > 0.9) and (avg_c < 1.1)
+
+    @patch('mindsdb.integrations.handlers.postgres_handler.Handler')
+    def test_describe_model(self, mock_handler):
+        self.set_handler(mock_handler, name='pg', tables={'df': self.df})
+
+        # create predictor
+        self.run_sql('''
+            create model proj.modelx
+            from pg (select * from df)
+            predict c
+            using
+                engine='autogluon',
+                tag = 'test_model';
+            ''')
+        self.wait_predictor('proj', 'modelx')
+
+        # run predict
+        ret = self.run_sql('''
+            DESCRIBE MODEL proj.modelx.model
+            ''')
+        # value is greater than 0, since atleast one model has been evaluated
+        assert (len(ret) > 0)
+
+    @patch('mindsdb.integrations.handlers.postgres_handler.Handler')
+    def test_describe_feature(self, mock_handler):
+        self.set_handler(mock_handler, name='pg', tables={'df': self.df})
+
+        # create predictor
+        self.run_sql('''
+            create model proj.modelx
+            from pg (select * from df)
+            predict c
+            using
+                engine='autogluon',
+                tag = 'test_model';
+            ''')
+        self.wait_predictor('proj', 'modelx')
+
+        # run describe
+        ret = self.run_sql('''
+            DESCRIBE MODEL proj.modelx.features
+            ''')
+        # value is greater than 1, since atleast target and atleast one feature is used
+        assert (len(ret) > 1)
+
+    @patch('mindsdb.integrations.handlers.postgres_handler.Handler')
+    def test_describe_model_noattr(self, mock_handler):
+        self.set_handler(mock_handler, name='pg', tables={'df': self.df})
+
+        # create predictor
+        self.run_sql('''
+            create model proj.modelx
+            from pg (select * from df)
+            predict c
+            using
+                engine='autogluon',
+                tag = 'test_model';
+            ''')
+        self.wait_predictor('proj', 'modelx')
+
+        # run predict
+        ret = self.run_sql('''
+            DESCRIBE MODEL proj.modelx
```

Comment:
Probably good to assert that the expected columns are in the returned dataframe, too.